packages/front-end/components/Settings/SubscriptionInfo.tsx

```diff
@@ -53,16 +46,6 @@ export default function SubscriptionInfo() {
       <div className="col-md-12 mb-3">
         <strong>Number Of Seats:</strong> {quote?.activeAndInvitedUsers || 0}
       </div>
-      {quote && (
```

Comment:
Removing this per Jeremy's recommendation in the Slack thread. Contact me for a link if needed.